python/pyfunc-server/pyfuncserver/protocol/upi/server.py

```diff
@@ -19,6 +20,7 @@ def __init__(self, model: PyFuncModel):
         self._model = model
 
     def PredictValues(self, request, context):
+        time.sleep(10)
```

Comment:
Is this expected?

Reply:
no, it is leftover from testing, I'll remove this